File: php/URI/1036.php
<?php

$linha = explode(' ', readline());

$delta = ($linha[1]*$linha[1]) - (4*$linha[0]*$linha[2]);

if($delta < 0 || $linha[0] == 0)
{
    echo "Impossivel calcular\n";
}
else
{
    $r1 = (-$linha[1]+sqrt($delta))/(2*$linha[0]);
    $r2 = (-$linha[1]-sqrt($delta))/(2*$linha[0]);
    echo "R1 = ".number_format($r1,5, ".","", ). "\n";
    echo "R2 = ".number_format($r2,5, ".","", ). "\n";
}

?>

File: C/URI/1036.c
#include <stdio.h>
 
int main() {
 
 double a, b, c, r1, r2, delta;
        scanf("%lf %lf %lf", &a, &b, &c);
        delta = (b * b) - (4.0 * a * c);
        if (0 > delta || (2.0 * a) == 0.0)
        printf("Impossivel calcular\n");
        else{ 
        r1 = (-b + sqrt(delta)) / (2.0 * a);
        r2 = (-b - sqrt(delta)) / (2.0* a);
        printf("R1 = %.5lf\nR2 = %.5lf\n", r1, r2);
        }
    return 0;
}